cmd/get.go

```diff
@@ -167,19 +175,35 @@ func GetDeploySpecTable(specs []deploymentspec.DeploymentSpec) (string, []string
 			replicas = fmt.Sprint(spec.GetString("replicas"))
 		}
 
+		specValues := []interface{}{spec.Cluster(), spec.Environment(), spec.Name(), spec.Version(), replicas, spec.GetString("type"), spec.GetString("deployStrategy/type")}
+		if releaseToDefined {
+			specValues = append(specValues, spec.GetString("releaseTo"))
+		}
+
 		row := fmt.Sprintf(
 			pattern,
-			spec.Cluster(),
-			spec.Environment(),
-			spec.Name(),
-			spec.Version(),
-			replicas,
-			spec.GetString("type"),
-			spec.GetString("deployStrategy/type"),
+			specValues...,
 		)
 		rows = append(rows, row)
 	}
-	return header, rows
+	return strings.Join(headers, "\t"), rows
+}
+
+func checkForReleaseToInSpecs(specs []deploymentspec.DeploymentSpec) bool {
+	releaseToDefined := false
+	for _, spec := range specs {
+		if spec.GetString("releaseTo") != "-" {
```

Comment:
Denne testen bør flyttes til package deploymentspec. Foreslår en ny metode hasReleaseTo() med boolean returverdi.

Reply:
laget en ny metode HasValue i deployspec.go som blir loopet i get.go for å se om releaseTo har verdi.